qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -174,13 +179,26 @@ def write_local_config(self):
             export TEST_DIR={}
             export SCRATCH_DEV={}
             export SCRATCH_MNT={}
-            export CEPHFS_MOUNT_OPTIONS="-o name=admin,secret={}"
+            export CEPHFS_MOUNT_OPTIONS="-o {}name=admin,secret={}"
             ''').format(self.test_dev, self.test_dirs_mount_path, self.scratch_dev,
-                        self.scratch_dirs_mount_path, self.get_admin_key())
+                        self.scratch_dirs_mount_path, _options, self.get_admin_key())
 
         self.mount_a.client_remote.write_file(join(self.xfstests_repo_path, 'local.config'),
                                               xfstests_config_contents, sudo=True)
 
+    def write_ceph_exclude(self):
+        from os.path import join
+        from textwrap import dedent
+
+        # These tests will fail or take to much time and will
+        # make the test timedout, just skip them for now.
+        xfstests_exclude_contents = dedent('''\
+            {c}/001 {g}/003 {g}/538 {g}/531
+            ''').format(g="generic", c="ceph")
+
```

Comment:
nit: This way it'll be more readable.

```suggestion
        xfstests_exclude_contents = dedent(f'''\
            {ceph}/001 {generic}/003 {generic}/538 {generic}/531
            ''')

```

Reply:
There could be many other test cases will be added later soon and then that will make the code less readable and not clean and tidy.